### PR TITLE
Manual: Extending chapter 28.3 'Testing the Exit Code'

### PR DESCRIPTION
**Pull request by deining**
Can you do me a favor and regenerate the html docs, please? Thanks.

### CODE REVIEW

**Comment by remkop** [comment]
Hi @deining, I’ll try to get to this in the next few days. 

I was planning to publish the manual with the next release. 
The only thing outstanding for the next release is #1322 (and that’s not really a showstopper, it’s just something I’d like to get out of the way, and doing a release is the best/only way to test it...)

File: docs/index.adoc [hunk 1]
```diff
@@ -11999,12 +11999,17 @@ class TestOutput {
 
 === Testing the Exit Code
 Testing the exit code of applications that call `System.exit` can be tricky but is not impossible.
+Depending on your programming language, your environment and your testing framemork, you have several options:
```
Comment (remkop) [inline docs/index.adoc:12002]: framemork -> framework

File: docs/index.adoc [hunk 2]
```diff
@@ -11999,12 +11999,17 @@ class TestOutput {
...
 
-==== Java 8+ and `catchSystemExit`
-Applications that use JUnit 5 or Java 8+ can use Stephan Birkner's https://github.com/stefanbirkner/system-lambda[System-Lambda] project for testing the https://github.com/stefanbirkner/system-lambda#systemexit[exit code].
-For example:
+Java 8:: You can use Stephan Birkner's https://github.com/stefanbirkner/system-lambda[System-Lambda] project for testing the https://github.com/stefanbirkner/system-lambda#systemexit[exit code]
+Java/Kotlin + Junit 5:: You can use Todd Ginsberg's https://github.com/tginsberg/junit5-system-exit[System.exit()] extension for Junit 5. The project README mentions common https://github.com/tginsberg/junit5-system-exit#use-cases[use cases].
+Java + Junit 4:: You can use Stephan Birkner's https://stefanbirkner.github.io/system-rules[System-Rules] project for testing the https://stefanbirkner.github.io/system-rules/#ExpectedSystemExit[exit code].
+Kotlin + Kotest:: Kotlin coders you may use the https://kotest.io/[Kotest] test framework which comes with https://kotest.io/docs/extensions/system_extensions.html#system-exit-extensions[System Exit extensions] that allow you to test the exit code.
```
Comment (remkop) [inline docs/index.adoc:12007]: I have not seen the convention to use two colon characters (`Java 8::`, `Java/Kotlin + Junit 5::`, etc) before. Is this an AsciiDoc thing that I am not aware of maybe?

Would it make sense to make this a bullet point list? That makes it easy to digest for readers, I think.

> * Java 8: You can use ...
> * Java/Kotlin + Junit 5: You can use...
> * Java + Junit 4: You can use...
> * Kotlin + Kotest: Kotlin coders you may use...

**Review by remkop — COMMENTED** [review]
Thanks for these improvements! Good to know there are lots of options for testing! :-) 

I added some comments.

File: docs/index.adoc (hunk 1 above)
Comment (deining) [reply docs/index.adoc]: Fixed.

File: docs/index.adoc (hunk 2 above)
Comment (deining) [reply docs/index.adoc]: >  Is this an AsciiDoc thing that I am not aware of maybe?

Yes, it's markup for [description lists](https://docs.asciidoctor.org/asciidoc/latest/lists/description/).

> Would it make sense to make this a bullet point list?

I liked the markup, but a bullet list is pretty much equivalent. I changed it to a bullet list.

**Review by remkop — APPROVED** [review]
Looks good, thank you!

Thanks also for explaining about the [description lists](https://docs.asciidoctor.org/asciidoc/latest/lists/description/). I didn't know that one. :-)

I will merge this now, and update the release notes later.
This will be published when I get around to doing the release. (Still looking at #1322).

Many thanks!

**Pull request merged**